.clang-format

```diff
@@ -0,0 +1,7 @@
+# Configures clang-format for the project.
+# Options can be consulted on https://clang.llvm.org/docs/ClangFormatStyleOptions.html
+BasedOnStyle: Google
+IndentWidth: 4
+ColumnLimit: 120
+TabWidth: 4
+UseTab: Never
```

Comment:
These config values are up for debate. Let me know what you guys think about it.